share/p2p/peers/manager.go

```diff
@@ -27,8 +27,6 @@ import (
 const (
 	// ResultNoop indicates operation was successful and no extra action is required
 	ResultNoop result = "result_noop"
-	// ResultSynced will save the status of pool as "synced" and will remove peers from it
-	ResultSynced = "result_synced"
```

Comment:
Do I understand correctly that we don't need it because we now clean up things quickly, and there is no point in keeping synced pools?

Reply:
Yeah, tracking synced / unsynced pools not worth the complexity anymore, as there won't be as many pools now